specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -1146,12 +1146,12 @@
       "properties": {
         "name": {
           "type": "string",
-          "readOnly": true,
```

Comment:
May I ask whether here "name" property under "latestAssignmentReport" property is the same thing with the "version" property under "[guestConfiguration](https://github.com/Azure/azure-rest-api-specs/blob/4bab0bfdf7b1e2e3a50eeda420c2891d087d1d85/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L797)" property?